Update troubleshoot.md (#3471)
MicrosoftDocs/memdocs: memdocs/configmgr/tenant-attach/troubleshoot.md

### <a name="bkmk_noauth"></a> Unauthorized to perform client action

> If the account that is logged into the Endpoint Admin Center doesn't have the required permissions in Configuration Manager, you'll see an `Unauthorized` response in the **CMGatewayNotificationWorker.log**.

Reviewer: ```suggestion
If the account that is logged into the Endpoint Manager admin center does not have the required permissions in Configuration Manager, you'll see an `Unauthorized` response in the **CMGatewayNotificationWorker.log**.
```

## Known issues

### Unhealthy Connector

> When you view the Microsoft Endpoint Configuration Maanger Connector in the Endpoint Admin Center > **Tenant Administration** > **Connectors and tokens** and it is showing as unhealthy or unknown you can delete the connector and this will sometimes resolve the issue. You will need to wait a couple of hours for the connector to be deleted and then a new connector should automatically be created. Do not change any of the Teanant Attach condfiguration in the Configuraton Manager Console.

Reviewer: ```suggestion
When you view the Microsoft Endpoint Configuration Manager connector in the Endpoint Manager admin center > **Tenant Administration** > **Connectors and tokens** and it shows as unhealthy or unknown, you could delete the connector to attempt to resolve the issue. You will need to wait a couple of hours for the connector to be deleted and for a new connector to be created automatically. Do not change any of the Tenant Attach configurations in the Configuration Manager console.
```

Author: good for me
